### PR TITLE
[EXP-3222] Expire cash Node SDK

### PR DESCRIPTION
**Pull request by Fenho**
## Summary
- Add cash payment support with field mapping from `payment_method` to `payment_type`
- Add payment intent expiration functionality for cash payments

## Changes
- Enhanced `PaymentIntentsManager` with field transformation and expire method
- Added TypeScript types for payment methods
- Added comprehensive tests and documentation

## Test plan
- [x] All existing tests pass (172 tests)
- [x] Cash payment creation and expiration functionality working
- [x] Field mapping validation and edge cases covered

🤖 Generated with [Claude Code](https://claude.ai/code)

### CODE REVIEW

File: src/lib/managers/paymentIntentsManager.ts [hunk 1]
```diff
@@ -3,5 +3,10 @@ import { PaymentIntent } from '../resources/paymentIntent';
 
 export class PaymentIntentsManager extends ManagerMixin<PaymentIntent> {
   static resource = 'payment_intent';
-  static methods = ['list', 'get', 'create'];
+  static methods = ['list', 'get', 'create', 'expire'];
+
+  expire(paymentIntentId: string): Promise<PaymentIntent> {
+    const path = `${this.buildPath()}/${paymentIntentId}/expire`;
+    return this._create({ path_: path });
+  }
```
[the remark this answers is not in the document]
Comment (Fenho) [reply src/lib/managers/paymentIntentsManager.ts:11]: 😮 wena
